### PR TITLE
Feature/tecg 175 order technology

### PR DESCRIPTION
**Pull request by felipeaziz**

### CODE REVIEW

File: src/main/java/com/ciandt/techgallery/service/TechnologyServiceImpl.java [hunk 1]
```diff
@@ -32,9 +39,14 @@
  */
 public class TechnologyServiceImpl implements TechnologyService {
 
+  private static final String NO_TECHNOLOGY_WAS_FOUND = "No technology was found.";
```
Comment (larruda) [inline src/main/java/com/ciandt/techgallery/service/TechnologyServiceImpl.java]: @felipeaziz there is a Enum for these messages.

File: src/main/java/com/ciandt/techgallery/service/endpoint/TechnologyEndpoint.java [hunk 2]
```diff
@@ -64,15 +68,20 @@ public Response getTechnologies() throws InternalServerErrorException, NotFoundE
...
-          throws ServiceException {
-    return service.findTechnologiesByFilter(
-        new TechnologyFilter(titleContains, shortDescriptionContains, recommendationIs), user);
+      @Named("recommendationIs") String recommendationIs,
+      @Named("orderOptionIs") String orderOptionIs) throws ServiceException {
+    return service.findTechnologiesByFilter(new TechnologyFilter(titleContains,
+        shortDescriptionContains, recommendationIs, orderOptionIs), user);
+  }
+
+  @ApiMethod(name = "getOrderOptions", path = "technology-getOrderOptions", httpMethod = "post")
```
Comment (larruda) [inline src/main/java/com/ciandt/techgallery/service/endpoint/TechnologyEndpoint.java]: @felipeaziz please do not use verbs on resources names. HTTP verbs should be used instead.

File: src/main/java/com/ciandt/techgallery/service/enums/TechnologyOrderOptionEnum.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+/**
+ * Enum for mapping Order Options.
+ * 
+ * @author Felipe Ibrahim
+ *
+ */
+public enum TechnologyOrderOptionEnum {
+
+  POSITIVE_RECOMENDATION_QUANTITY("Positive Recomendations Quantity"),
```
Comment (larruda) [inline src/main/java/com/ciandt/techgallery/service/enums/TechnologyOrderOptionEnum.java]: @felipeaziz same as above. There is a place for this already.

Comment (felipeaziz) [inline src/main/java/com/ciandt/techgallery/service/enums/TechnologyOrderOptionEnum.java]: This Enum is used to populate the combobox with order options in the front end. So I create in a separated Enum.

**Pull request merged**